Makefile

```diff
@@ -12,15 +12,12 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-ifneq (, $(BUILDX_BIN))
-	export BUILDX_CMD = $(BUILDX_BIN)
-else ifneq (, $(shell docker buildx version))
-	export BUILDX_CMD = docker buildx
-else ifneq (, $(shell which buildx))
-	export BUILDX_CMD = $(which buildx)
-else
-	$(error "Buildx is required: https://github.com/docker/buildx#installing")
-endif
```

Comment:
Rather than using the funky `buildx-cmd` target with the eval function below, could we instead change the `else` block of the above code to be

```Makefile
else
    export BUILDX_CMD = echo "Buildx is required: https://github.com/docker/buildx#installing"; false
```